File: birthdayCakeCandles.js
/* Birthday Candles
count how many items in
an array are the highest value */
candles = [1, 2, 3, , 3, 4]

function birthdayCakeCandles(candles) {
    high = 0
    highCount = 0

    for (let i = 0; i < candles.length; i++) {
        if (candles[i] > high) {
            high = candles[i]
        }
    }
    for (let i = 0; i < candles.length; i++) {
        if (candles[i] === high) {
            highCount += 1
        }
    }
    return (highCount)
}

console.log(birthdayCakeCandles(candles))

File: birthday_cake_candles.py
# BIRTHDAY CAKE CANDLES #
# return the the largest int in the list and count

candles = [4, 4, 1, 2]


def birthday_cake_candles(candles):
    tallest_candle = 0
    for candle in candles:
        if candle > tallest_candle:
            tallest_candle = candle

    return(tallest_candle, candles.count(tallest_candle))


print(birthday_cake_candles(candles))
